content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -108,6 +108,10 @@ If you haven't already set up a cluster locally, run `minikube start` to create
    http://172.17.0.15:31637
    ```
 
+   ```shell
+   curl ip-addr
```

Comment:
```suggestion
   curl http://172.17.0.15:31637
```

Reply:
I am a little bit confused about the output for this `curl ip-addr` is this curl http://172.17.0.15:31637 can you please tell me it will be helpful for me?

Reply:
This task is about exposing a service and accessing it using the `curl` command.
The `minikube service <service> --url` command prints the service URL.
The next command obviously should be about accessing that URL using `curl`.
Putting `ip-addr` is more confusing than solving an existing issue, right?